core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowKeys.scala

```diff
@@ -51,6 +52,7 @@ trait CloudflowTaskKeys {
   val cloudflowDockerImageName = taskKey[Option[DockerImageName]]("The name of the Docker image to publish.")
   val cloudflowDockerRegistry = taskKey[Option[String]]("The hostname and (optional) port of the Docker registry to use.")
   val cloudflowDockerRepository = taskKey[Option[String]]("The image repository name on the Docker registry.")
+  val cloudflowFlinkDockerImageName = taskKey[Option[DockerImageName]]("The name of the Flink Docker image to publish.")
```

Comment:
Why do we need this key?

Reply:
It's the key for the name of the image we build

Reply:
Does it have to be a key I mean.